pkgs/by-name/ke/keylight-cli/package.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  babashka,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "keylight-cli";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "versality";
+    repo = "keylight-cli";
+    rev = "46ff8472f9544e16eea822bf274acf4a5cbedbe8";
```

```suggestion
    tag = "v${version}";
```
Since you added a tag, we can use it here :)

---

done. thank you.